scripts/code_style.py

```diff
@@ -51,6 +51,12 @@ def list_generated_files() -> FrozenSet[str]:
     checks = re.findall(CHECK_CALL_RE, content)
     return frozenset(word for s in checks for word in s.split())
 
+# Check for comment string indicating an auto-generated file
+AUTOGEN_RE = re.compile(r"Warning[ :-]+This file is (now )?auto[ -]generated", re.ASCII | re.IGNORECASE)
```

Comment:
Just a minor relaxation in case someone as bad at spelling as I am touches the generation code.
```suggestion
AUTOGEN_RE = re.compile(r"Warning[ :-]+This file is (now )?auto[ -]?generated", re.ASCII | re.IGNORECASE)
```